config/examples/Creality/Ender-3/BigTreeTech SKR E3-DIP V1.1/Configuration.h

```diff
@@ -1645,7 +1636,7 @@
  *
  * View the current statistics with M78.
  */
-//#define PRINTCOUNTER
+#define PRINTCOUNTER
```

Comment:
I highly discourage the use of PRINTCOUNTER with anything emulating eeprom in flash. It is slow and is likely undesirable during a print. It also reduces the life of the flash chip, which is only rated for 10,000 erase cycles. Just over a year of continuous printing could render it useless.

Reply:
Agreed. I will roll back this change.